packages/fiori/src/ViewSettingsDialog.ts

```diff
@@ -202,6 +203,26 @@ class ViewSettingsDialog extends UI5Element {
 	@property({ type: Boolean })
 	sortDescending!: boolean;
 
+	/**
+	 * Defines the ID or DOM Reference of the element at which the menu is shown.
```

Comment:
instead of "menu" it should be "dialog" or "settings dialog". And the `@since` tag to reference 2.0 version.

Reply:
Yes, documentation will be fixed, this PR was uploaded before it was ready for review. Will request it after it's good to go.